test/index.test.ts

```diff
@@ -101,4 +98,41 @@ describe('Recover the address function of a transaction', () => {
       expect(recoveredAddress).not.toBe(signingAddress);
     });
   });
+
+  it('should recover the adress even if the message is an object', () => {
+    const lsp6Signer = new LSP6Signer();
+    testCases.forEach((data) => {
+      const messageData = lsp6Signer.sign(data, signingKey);
+      const signature = messageData.signature;
+      const isMessagePrefixed = true;
+      const recoveredAddress = lsp6Signer.recover(
+        messageData,
+        signature,
+        isMessagePrefixed,
+      );
+      expect(recoveredAddress).toBe(signingAddress);
+    });
+  });
+
+  it('should be prefixed with "\x19LSP6 Execute Relay Call:\n"', () => {
```

Comment:
this is more like testing 'should accept both hashed and unhashed message'

Reply:
If i read this in the title, I would expect to see a test that has "expect LSP6 Execute Relay Call..." somewhere. I don't see it.